### DISCUSSION TITLE
Light Mode

### DISCUSSION THREAD

**Discussion by xaryangupta** [darkreader/darkreader#6720] [Ideas]
Just like we can DarkMode any site which isn't darkmode, can we lightmode any website that isn't lightmode like YoutubeMusic&Spoitfy? This idea can be extrapolated to themeing with light green/blue/pink, which looks good too :)

**Comment by Gusted** [comment]
I mean, it's possible but not recommended nor actually supported. You can simply change the "Theme" to light, and everything will be parsed according to the light colors and few corrections for the color algorithm to help with this.

**Reply by xaryangupta** [reply]
but it doesn't work, try youtube music or spotify, they both have broken light themes

**Reply by Gusted** [reply]
Like I said, it's not supported to create a light color scheme. So if it doesn't work then it doesn't work.

**Reply by xaryangupta** [reply]
So is it not even in the plans?

**Reply by Gusted** [reply]
Nope, not even sure why it should be a goal of **dark** reader.

**Reply by real-felix** [reply]
@Gusted Is there still no plan to support it? The light mode works well, but:
- It also applies to websites already in light mode, so it should first check that it doesn't use a light theme;
- I have for some reason to enable it on each *dark* website individually, but the theme applies to light ones, so it's the opposite of what I want.

That would be so cool if it was supported, I have crazy eyestrain issues with black backgrounds, and I cannot find any accessibility extension to fix that. This one is the closest, and it works quite well.
